KlassifizierungQBert.py

```diff
@@ -0,0 +1,266 @@
+    plt.show()
+    pix1 = qbert.crop((35, 145, 50, 160)).load()
+    qBertColour1 = pix1[7, 8]
+    print(qBertColour1)
+
+    qbert = Image.open('/media/annika/Daten/BA-Datensaetze/atari_v2_release/screens/qbert/141/2234.png')
+    plt.imshow(qbert)
+    plt.show()
+    plt.imshow(qbert.crop((75, 70, 92, 90)))
+    plt.show()
+    pix2 = qbert.crop((75, 70, 92, 90)).load()
+    qBertColour2 = pix2[8, 11]
+    print(qBertColour2)
+
+    qbert = Image.open('/media/annika/Daten/BA-Datensaetze/atari_v2_release/screens/qbert/141/935.png')
+    plt.imshow(qbert)
+    plt.show()
+    plt.imshow(qbert.crop((60, 100, 75, 125)))
+    plt.show()
+    pix3 = qbert.crop((60, 100, 75, 125)).load()
+    qBertColour3 = pix3[6, 13]
+    print(qBertColour3)
+
+"""
+Funktion generiert einen Namen für das Bild:
+    colourClass: Sagt aus, welche Farbklasse ein Bild erhält, abhängig von der Pyramidenfarbe in (r,g,b)
+    qBert:       False, wenn Q*Bert nicht im Bild ist -> "NoQbert"
+                 True, wenn Q*bert im Bild ist -> "WithQbert"
+    Number:      Nummer des Bildes
+"""
+def createNameImg(colourClass,qBert,number):
+    if qBert:
+        return '{}_({},{},{})_WithQbert.png'.format(number, colourClass[0], colourClass[1], colourClass[2])
+    else:
+        return '{}_({},{},{})_NoQbert.png'.format(number,colourClass[0],colourClass[1],colourClass[2])
+
+"""
+Funktion generiert einen Namen für den Ordner:
+    colourClass: Sagt aus, welche Farbklasse ein Bild erhält, abhängig von der Pyramidenfarbe in (r,g,b)
+    qBert:       False, wenn Q*Bert nicht im Bild ist -> "NoQbert"
+                 True, wenn Q*bert im Bild ist -> "WithQbert"
+    Number:      Nummer des Bildes
+"""
+def createNameFolder(colourClass,qBert):
+    if qBert:
+        return 'ColourClass:({},{},{})__WithQbert.png'.format(colourClass[0], colourClass[1], colourClass[2])
+
+    else:
+        return 'ColourClass:({},{},{})_NoQbert'.format(colourClass[0],colourClass[1],colourClass[2])
+"""
+Funktion generiert einen neuen Ordner mit dem gegebenen Pfad
+"""
+def createFolder(folderpath):
+    if not os.path.exists(folderpath):
+        os.makedirs(folderpath)
+
+"""
+Start der Klassifizierung.
+Grundlage ist der Datensatz von der Webseite: http://atarigrandchallenge.com/data
+"""
+if __name__ == '__main__':
+    """training_data = []
+    create_training_data('/media/annika/Daten/BA-Datensaetze/atari_v2_release/screens/qbert/7')
+
+    print(len(training_data))
+    print(training_data[0].shape)
+
+    save_dataset('/home/annika/BA-Datensaetze/datasetQBertFolder7.npy', training_data)
+    """
+    global boxColoursAndQuantity
+    boxColoursAndQuantity = []
+    global qBertQuantity
+    qBertQuantity = 0
+    #countNumberOfDifferentColours('/media/annika/Daten/BA-Datensaetze/atari_v2_release/screens/qbert/141')
+
+    pathQBertPicFolder = '/media/annika/Daten/BA-Datensaetze/atari_v2_release/screens/qbert'
+    pathNewQBertDataset = '/media/annika/Daten/QBert_Datensatz'
```

Comment:
Die Pfade müssten geändert werden, da dass nur auf deinem Rechner geht. Am besten du schreibst in die README einen "Install" Teil in dem erklärt wird was benötigt wird und wohin es gedownloaed werden soll. Du könntest z.B. einen Unterordner "datasets" erstellen und hier wird dann nur realtiv referenziert:

```suggestion
    pathQBertPicFolder = 'datasets/atari_v2_release/screens/qbert'
    pathNewQBertDataset = 'datasets/QBert_Datensatz'
```